app/Controllers/Reports.php

```
@@ -1884,6 +1898,57 @@ public function detailed_sales(string $start_date, string $end_date, string $sal
         return view('reports/tabular_details', $data);
     }
 
+    public function detailed_item_sales(string $start_date, string $end_date, string $sale_type = '0', string $location_id = 'all'): string
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$sale_type'. (undefined)

(UnusedFormalParameter)